src/main/java/org/folio/linked/data/service/impl/ResourceServiceImpl.java

```diff
@@ -254,4 +257,17 @@ private void breakEdgesAndDelete(Resource resource) {
     breakCircularEdges(resource);
     resourceRepo.delete(resource);
   }
+
+  private void addExistingEdges(Resource resource) {
```

Comment:
Let us rename this method to "addIncomingEdgesForWork" to make the intention clear.

Reply:
done